manifests/init.pp

```diff
@@ -52,6 +68,14 @@
   Boolean $pulp_master = $foreman_proxy_content::params::pulp_master,
   String $pulp_admin_password = $foreman_proxy_content::params::pulp_admin_password,
   Optional[String] $pulp_max_speed = $foreman_proxy_content::params::pulp_max_speed,
+  Optional[Integer[1,]] $pulp_max_tasks_per_child = $foreman_proxy_content::params::pulp_max_tasks_per_child,
```

Comment:
I'm surprised the trailing comma works but I think it's cleaner without in this case.